2019 09 cm pb 68

rquevedo opened the pull request.

cmanuelalvarez11 committed:
[IMP] auth_brute_force: Traducciones
[IMP] auth_brute_force: Llevamos a 20 el numero de intentos
[IMP] auth_brute_force: Modificaciones para mostrar el mensaje de bloqueo

Change-Id: I90a4afecda9c4bf9180413a9144b53b47d2a6438

cmanuelalvarez11 committed:
[IMP] base_export_security: Traducciones
[IMP] base_export_security: Mejorando mensaje de error

Change-Id: I0bc8a07d859a61f0edeaab7a72c289cd5456d2b3

rquevedo committed:
[IMP] base_import_security_group: Mejora de la traducción

Change-Id: I6e48f832598373fa5992915cbbd736224cb95f90

pepetreshere commented on base_export_security/i18n/es.po:
```diff
@@ -142,23 +178,45 @@ msgstr "Última actualización por"
+
+#. module: base_export_security
+#: model:ir.model.fields,field_description:base_export_security.field_account_bank_statement_import_journal_creation_margin_right
+msgid "Right Margin (mm)"
```
estas cosas son de otros modulos!

rquevedo replied:
ok

pepetreshere commented on base_export_security/i18n/es.po:
```diff
@@ -53,23 +53,59 @@ msgstr "Creado por"
+#. module: base_export_security
+#: model:ir.model.fields,help:base_export_security.field_account_bank_statement_import_journal_creation_company_mnemonic
+msgid "Este código va ser usado para generar el cash management del banco bolivariano, debe tener 6 digitos."
+msgstr "Este código va ser usado para generar el cash management del banco bolivariano, debe tener 6 digitos."
```
estas cosas son de otros modulos

rquevedo replied:
ok

rquevedo committed:
[FIX] base_export_security: Mejoras a las traducciones

Change-Id: Iea76c1e9459a21cb888431bbafefa777d2d40190

pepetreshere commented on auth_brute_force/controllers/__init__.py:
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from . import main
```
Hagamos un extra en este módulo, en el form de usuario agregar (programar en el mismo módulo de la OCA):
- botón para desbloquear el usuario (o expliquenme como desbloqueo)
- botón para ver intentos de login (modelo res.authentication.attempt), ojo ya existe el menú

rquevedo replied:
Hecho

rquevedo committed:
[IMP] auyh_brute_force: Nuevo botón para desbloquear usuario
[IMP] auyh_brute_force: Nuevo botón para ver registros de inicio de sesión del usuario

Change-Id: I7cd5306c1c3fcbb26b2604db9b7a1f56af19d5bd

rquevedo committed:
[FIX] base_export_security: Se asigna una categoría común a la importación y exportación de csv

Change-Id: I157325ed8e98a0e1f3be4dff792e5954f6896693

rquevedo committed:
[FIX] base_import_security_group: Se asigna una categoría común a la importación y exportación de csv

Change-Id: Ic31f051d702625c8414176e5f691cbea3a68cfc6
